frontend/src/app/lib/InstallManager.js

```diff
@@ -15,13 +15,29 @@ if (typeof navigator !== 'undefined') {
   mam = navigator.mozAddonManager;
 }
 
-function mozAddonManagerInstall(url) {
+function mozAddonManagerInstall(url, sendToGA) {
   return mam.createInstall({ url }).then(install => {
     return new Promise((resolve, reject) => {
```

Comment:
I think this should be removed, and the "start", "end", "experimentTitle" and "install.install().then(..." lines moved up a level of indentation.